docs/process/releasing-updates.md

```diff
@@ -6,25 +6,60 @@ We have three channels to which we can release: `production`, `beta`, and `test`
 
 - `production` is the channel from which the general public downloads and receives updates. It should be stable and polished.
 
-- `beta` is released more often than `production`. It may be buggy and unpolished.
+- `beta` is released more often than `production`. We want to ensure `master` is always in a state where it can be released to users and we will use `beta` for additional QA.
 
 - `test` is unlike the other two. It does not receive updates. Each test release is locked in time. It's used entirely for providing test releases.
 
 ## The Process
 
-From a clean working directory, set the `GITHUB_ACCESS_TOKEN` environment variable to a valid [Personal Access Token](https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/) and run:
+### 1. GitHub Access Token
+
+From a clean working directory, set the `GITHUB_ACCESS_TOKEN` environment variable to a valid [Personal Access Token](https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/) 
+
+You can check that this is set on macOS by:
+```shellsession
+$ env GITHUB_ACCESS_TOKEN
+```
+
+You can check that this is set on Windows by:
+```shellsession
+$ echo %GITHUB_ACCESS_TOKEN%
+```
+
+If you are creating a new Personal Access Token on GitHub:
+* make the token memorable - use a description like `Desktop Draft Release and Changelog Generator`
+* the `read:org` scope is the **only** required scope for drafting releases
+
+You can add an access token as an environment variable on macOS by: 
+```shellsession
+$ export GITHUB_ACCESS_TOKEN={your token here}
+```
+
+You can add an access token as an environment variable on Windows by: 
```

Comment:
Similar to above, this is specific to Command Prompt. `export ...` works fine in Git Bash, and PowerShell supports `$env:GITHUB_ACCESS_TOKEN="{your token here}"` (the quote marks are important).

Reply:
I should have been clearer here about organizing things per-shell, so I just pushed 43e48f9c76c2485ce6ce421e67b20e8fed15e5c0 to do that. Let me know if that makes sense to you too.